src/extensions/electrodegeometry/ElectrodeGeometryWidget/electrodeGeometryLayer.ts

```diff
@@ -0,0 +1,266 @@
+import { norm } from "mathjs"
```

Comment:
Separate out the electrodeGeometryLayer into its own .ts file so it can be reused.